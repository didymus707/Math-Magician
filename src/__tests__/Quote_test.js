import React from 'react';
import TestUtils from 'react-dom/test-utils';
import { BrowserRouter, Route } from 'react-router-dom';
import Quote from '../components/Home';

describe('Home', () => {
  it('wraps content within a div with class container', () => {
    const quote = TestUtils.renderIntoDocument(
      <BrowserRouter>
        <Route exact path="/" component={Quote} />
      </BrowserRouter>,
    );
    TestUtils.findRenderedDOMComponentWithClass(
      quote,
      'quo',
    );
  });
});
