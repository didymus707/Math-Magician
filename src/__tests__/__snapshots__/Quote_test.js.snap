// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders correctly 1`] = `
Array [
  <nav
    className="nav-con"
  >
    <h1>
      Math Magician
    </h1>
    <ul>
      <li
        className="home"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="calc"
      >
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calulator
        </a>
      </li>
      <li
        className="quo"
      >
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>,
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </p>,
]
`;
